src/FixedDataTableCell.js

```diff
@@ -210,6 +187,7 @@ class FixedDataTableCell extends React.Component {
     var style = {
       height,
       width,
+      overflow: 'visible',
```

Comment:
I suspect this was done to render the resizer line inside the cell's component's DOM.
But won't this break existing applications? Is there another way to solve this?

Reply:
Yeah, Removed